Spring-server/src/main/java/ikpi63holding/postclient/PostMessage.java

```diff
@@ -1,20 +1,15 @@
 package ikpi63holding.postclient;
 
+import java.io.File;
+import java.io.FileOutputStream;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import javax.mail.Address;
-import javax.mail.BodyPart;
-import javax.mail.Message;
-import javax.mail.MessagingException;
-import javax.mail.Session;
-import javax.mail.internet.AddressException;
-import javax.mail.internet.InternetAddress;
-import javax.mail.internet.MimeMessage;
-import javax.mail.internet.MimeMultipart;
+import java.io.InputStream;
+import java.util.*;
+import javax.mail.*;
+import javax.mail.internet.*;
```

Comment:
Same thing with wildcards

Reply:
@Daniel1898 you have to use full path to insert only necessary imports